packages/components/src/atoms/Select/Select.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { forwardRef } from 'react'
+import type { SelectHTMLAttributes } from 'react'
+import { Icon } from '../..'
+import { CaretDown } from '@faststore/ui/src/assets'
+
+export interface SelectProps extends SelectHTMLAttributes<HTMLSelectElement> {
+  /**
+   * Redefines the id property to be required when using the Select component. The
+   * id will be used to link the UISelect component and its label.
+   */
+  id: string
+  /**
+   * Defines the options available in the select. The SelectOptions object
+   * keys are the property names, while the values correspond to the text that
+   * will be displayed in the UI.
+   */
+  options: Record<string, string>
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+}
+
+const Select = forwardRef<HTMLSelectElement, SelectProps>(function Select(
+  { options, id, testId = 'fs-select', ...otherProps }: SelectProps,
+  ref
+) {
+  return (
+    <div data-fs-select>
+      <select id={id} ref={ref} data-testid={testId} {...otherProps}>
+        {Object.keys(options).map((key) => (
+          <option key={key} value={key}>
+            {options[key]}
+          </option>
+        ))}
+      </select>
+      <Icon data-fs-select-icon component={<CaretDown />}/>
```

Comment:
Should we allow the user to change this component? (In this case, I think it's better to make this icon as default and doesn't receive any icon as prop) 👍